app/controllers/doctors_controller.rb

```diff
@@ -1,11 +1,50 @@
 class DoctorsController < ApplicationController
+  before_action :set_Doctor, only: [:edit, :update, :show]
```

Comment:
The following method `set-Doctor` is undefined when I am trying to access this path.
I suggest you could write `set_doctor` instead.